### PR TITLE
run shell scripts through shellcheck

### PR DESCRIPTION
**Pull request by neheb**
- [ ] References issue
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

**Review by votdev — CHANGES_REQUESTED** [review]
Could you please explain in detail why these changes are necessary.

**Comment by votdev** [comment]
https://cbea.ms/git-commit/ may be helpful to create a PR that helps to understand why these changes are done.

File: deb/openmediavault/etc/cron.daily/openmediavault-check_ssl_cert_expiry [hunk 1]
```diff
@@ -33,17 +33,17 @@ OMV_SSL_CERTIFICATE_CHECK_EXPIRY_ENABLED=${OMV_SSL_CERTIFICATE_CHECK_EXPIRY_ENAB
...
+for f in /etc/ssl/certs/"${OMV_SSL_CERTIFICATE_PREFIX}"*.crt ; do
   [ -e "${f}" ] || continue
 
   omv_syslog_info "Validating the SSL certificate ${f} ..."
 
   enddate=$(openssl x509 -enddate -noout -in "${f}" | sed "s/.*=\(.*\)/\1/" | awk -F " " '{print $1,$2,$3,$4}')
   enddate_seconds=$(date -d "${enddate}" +%s)
   now_seconds=$(date -d now +%s)
-  datediff_days=$(( (${enddate_seconds} - ${now_seconds}) / (60 * 60 * 24) ))
+  datediff_days=$(( (enddate_seconds - now_seconds) / (60 * 60 * 24) ))
```
[the remark this answers is not in the document]
Comment (neheb) [reply deb/openmediavault/etc/cron.daily/openmediavault-check_ssl_cert_expiry:44]: $ in (()) is not needed. https://www.shellcheck.net/wiki/SC2004

**Comment by neheb** [comment]
A lot of different checks are triggered:

https://www.shellcheck.net/wiki/SC2124
https://www.shellcheck.net/wiki/SC2068
https://www.shellcheck.net/wiki/SC2004
https://www.shellcheck.net/wiki/SC2046
https://www.shellcheck.net/wiki/SC2162
https://www.shellcheck.net/wiki/SC2143

**Comment by votdev** [comment]
Your commitment in honor, but I have the fear that the one or the other change has side effects and things will no longer work afterwards that have worked so far without problems.

I think we now need unit tests for the changed functions.

**Comment by votdev** [comment]
I still do not know your intention about this PR. Please file the PR description field why this PR is necessary.

The code has been used for years without issues now. I am critical of changing the code just for the sake of beauty.

Please split the PR into separate PRs, one for the cron jobs that can be validated easily, and one for the helper functions. It is absolutely necessary to introduce unit tests with that PR, otherwise i will not accept it.

**Comment by neheb** [comment]
I was noticing issues with the check_btrfs cron scripts not finding a volume already mounted. fixed in latest version. So I decided to look at it with shellcheck. This is mostly a cleanup PR. I'll split this up.

**Comment by votdev** [comment]
Thx for the tips. Superseded by https://github.com/openmediavault/openmediavault/pull/1525.

**Pull request closed without merging**